i18n/README-PL.md

```diff
@@ -1,107 +1,162 @@
+Silnik gry oparty na Flutter.
 </p>
 
 <p align="center">
-  <a title="Pub" href="https://pub.dev/packages/flame/versions#prerelease" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout&include_prereleases" /></a>
   <a title="Pub" href="https://pub.dev/packages/flame" ><img src="https://img.shields.io/pub/v/flame.svg?style=popout" /></a>
-  <img src="https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=main&event=push" alt="Test" />
+  <img src="https://github.com/flame-engine/flame/workflows/cicd/badge.svg?branch=main&event=push" alt="Test" />
   <a title="Discord" href="https://discord.gg/pxrBmy4" ><img src="https://img.shields.io/discord/509714518008528896.svg" /></a>
+  <a title="Melos" href="https://github.com/invertase/melos"><img src="https://img.shields.io/badge/maintained%20with-melos-f700ff.svg"/></a>
 </p>
 
 ---
 
-[English](/README.md) | [简体中文](/i18n/README-ZH.md) | [Polski](/i18n/README-PL.md) | [Русский](/i18n/README-RU.md) | [日本語](/i18n/README-JA.md)
+[English](/README.md) | [简体中文](/i18n/README-ZH.md) | [Polski](/i18n/README-PL.md) | [Русский](/i18n/README-RU.md) | [Español](/i18n/README-ES.md) | [日本語](/i18n/README-JA.md)
 
 ---
 
+
+## Dokumentacja
+
+Pełna dokumentacja Flame znajduje się na
+[docs.flame-engine.org](https://docs.flame-engine.org/).
+
+Aby zmienić wersję dokumentacji, użyj selektora wersji za pomocą `version:` na górze strony.
+
+**Uwaga**: Dokumentacja znajdująca się na gałęzi main jest nowsza, niż dokumentacja udostępniona na stronie docs.
+
+Inne przydatne linki:
+ - [Oficjalna strona Flame](https://flame-engine.org/).
+ - [Przykłady](https://examples.flame-engine.org/) większości funkcji, które można wypróbować z poziomu twojej przeglądarki.
+ - [Poradniki](https://tutorials.flame-engine.org/) - Prosty samouczek na początek.
+ - [Dokumentacja API](https://pub.dev/documentation/flame/latest/) - Wygenerowana dokumentacja API dartdoc.
+
+
 ## Pomoc
 
-Mamy kanał pomocy Flame na Discord Blue Fire, dołącz do niego [tutaj](https://discord.gg/5unKpdQD78).
+Społeczność Flame jest na [serwerze Discord Blue Fire's](https://discord.gg/5unKpdQD78), gdzie możesz zadać jakiekolwiek pytanie związane z Flame.
+
+Jeśli czujesz się bardziej komfortowo ze StackOverflow, możesz również utworzyć tam pytanie. Dodaj
+[tag Flame](https://stackoverflow.com/questions/tagged/flame), aby każdy, kto śledzi tag, mógł pomóc.
+
+
+## Cechy
+
+Celem Flame Engine jest dostarczenie kompletnego zestawu nietuzinkowych rozwiązań typowych problemów, które mogą być do siebie podobne dla gier opracowanych z Flutter.
+
+Oto niektóre z kluczowych dostępnych funkcji:
 
-Mamy też najczęściej zadawane pytania - [FAQ](FAQ.md), więc najpierw wyszukaj tam swoje pytanie.
+ - Pętla gry.
+ - System komponent/obiekt (FCS).
+ - Efekty i cząsteczki.
+ - Wykrywanie kolizji.
+ - Obsługa gestów i wprowadzania danych
+ - Obrazy, duszki (sprites) i arkusze duszka
+ - Ogólne narzędzia ułatwiające programowanie.
```

Comment:
Brace yourselves. Huge nitpick incoming 😅

Every of these points should probably end with a period, the same way they do in the English docs:

> - A game loop.
> - A component/object system (FCS).
> - Effects and particles.
> - Collision detection.
> - Gesture and input handling.
> - Images, animations, sprites, and sprite sheets.
> - General utilities to make development easier.

```suggestion
 - Pętla gry.
 - System komponent/obiekt (FCS).
 - Efekty i cząsteczki.
 - Wykrywanie kolizji.
 - Obsługa gestów i wprowadzania danych.
 - Obrazy, duszki (sprites) i arkusze duszka.
 - Ogólne narzędzia ułatwiające programowanie.
```

Reply:
Good point, thanks a lot! ✨